packages/salesforcedx-vscode-core/src/commands/sourceDiff.ts

```diff
@@ -109,7 +109,7 @@ export const handleCacheResults = async (
     }
   } else {
     const message = nls.localize('source_diff_components_not_in_org');
-    await notificationService.showErrorMessage(message);
+    notificationService.showErrorMessage(message);
```

Comment:
Please use `void notificationService.showErrorMessage(message);`

Reply:
Also, on line 108, the await is not necessary, given `differ.diffFolder` does not return a promise.

Reply:
good catch! done